scripts/third_party_licenses/other.tpl

```diff
@@ -0,0 +1,9 @@
+{{ range . -}}
+{{ if ne .LicenseName "Apache-2.0" -}}
+--------------------------------------------------------------------------------
+** {{.Name}}; version {{.Version}} - https://{{.Name}}
+
+{{ .LicenseText }}
```

Comment:
Can't this be one template file with the `{{ .LicenseText }}` being conditional on if its Apache 2.0 or not?

Reply:
The way this ends up in the license file is:
```
- Proj1 - info
- Proj2 - info
- Proj3 - info
Apache 2.0 license text
---
- Proj4 - info
Proj4 license text
---
- Proj5 - info
Proj5 license text
```

I couldn't figure out a way to do that with 1 template.